static-site/src/components/layout.jsx

```diff
@@ -23,6 +23,9 @@ export default class MainLayout extends React.Component {
           <title>{`${siteTitle}`}</title>
           <meta name="description" content={siteDescription} />
           <link rel="me" href="https://mstdn.science/@nextstrain" />
+          <link href={`${siteUrl}/blog/atom.xml`} rel="alternate" title="Atom feed for nextstrain.org/blog" type="application/atom+xml" />
+          <link href={`${siteUrl}/blog/feed.json`} rel="alternate" title="JSON feed for nextstrain.org/blog" type="application/json" />
+          <link href={`${siteUrl}/blog/rss2.xml`} rel="alternate" title="RSS2 feed for nextstrain.org/blog" type="application/rss+xml" />
         </Helmet>
```

Comment:
_non-blocking_

These href values could be more tighly coupled with where they are defined:

https://github.com/nextstrain/nextstrain.org/blob/5709c3ce11f60cf66fa4ae3604643018e70e2e00/static-site/src/util/blogPosts.js#L61-L65

They could be top-level constants in `blogPosts.js` and imported here.